UT4MasterServer.Web/src/pages/Servers/enums/game-mode.ts

```diff
@@ -0,0 +1,9 @@
+// This isn't a complete list, but it is only used in quick play for now and I think these are all the possible values for quick play
+export enum GameMode {
+  duel = '/Script/UnrealTournament.UTDuelGameMode',
+  deathmatch = '/Script/UnrealTournament.UTDMGameMode',
+  ctf = '/Script/UnrealTournament.UTCTFGameMode',
+  blitz = '/Script/UnrealTournament.UTFlagRunGame',
+  empty = 'EMPTY',
+  hub = '/Script/UnrealTournament.UTLobbyGameMode'
```

Comment:
Add all stock game modes:
siege = `/Script/UnrealTournament.UTFlagRunPvEGame`
team_deathmatch = `/Script/UnrealTournament.UTTeamDMGameMode`
showdown= `/Script/UnrealTournament.UTShowdownGame`

And commonly played custom game modes which will be possible to play in quickplay:
elimination = `/Game/ALTS/Elimination_113.Elimination_113_C`
bunnytrack = `/Game/BT/Gamemode/BT_Game.BT_Game_C` // not confirmed, may be wrong

Reply:
Added